### PR TITLE
Removes references to console screens and removes an unnecessary var from circuitboards

### PR DESCRIPTION
**Pull request by SteelSlayer**
## What Does This PR Do
Since console screens do not exist anymore, they shouldn't be referenced anywhere in the game.
This removes the variable `frame_desc` from circuitboards because it's not even used anywhere, *and* it contained reference to the console screens.

## Changelog
:cl:
spellcheck: Examine text for circuit boards that require more than 1 of a part will now have an "s" on the end of the name of that part. For example "2 capacitor" becomes "2 capacitors"
/:cl:

### CODE REVIEW

File: code/game/machinery/constructable_frame.dm [hunk 1]
```diff
@@ -881,7 +894,7 @@ to destroy them and players will be able to make replacements.
 	build_path = /obj/machinery/atmospherics/unary/cryo_cell
 	board_type = "machine"
 	origin_tech = "programming=4;biotech=3;engineering=4;plasmatech=3"
-	frame_desc = "Requires 1 Matter Bin, 1 piece of cable and 4 Console Screens."
+	frame_desc = "Requires 1 Matter Bin, 1 piece of cable and 4 Sheets of glasss."
```
Comment (Fox-McCloud) [inline code/game/machinery/constructable_frame.dm:897]: `glasss`?

**Comment by Fox-McCloud** [comment]
I'll be honest, I dunno why these aren't just automagically generated.

File: code/game/machinery/constructable_frame.dm (hunk 1 above)
Comment (SteelSlayer) [reply code/game/machinery/constructable_frame.dm:897]: I.. dont know how that happened :smile:

**Comment by SteelSlayer** [comment]
That would probably be a good idea.

**Comment by SteelSlayer** [comment]
I just realized we DO have these automatically generated in `/obj/item/circuitboard/examine`. And actually, `frame_desc` isn't even used so I've just removed the var completely. The only thing the examine text doesn't look good on is cables but, because its says "cable coils" instead of "pieces of cable", but not a huge deal I think

image

**Comment by Fox-McCloud** [comment]
Woooo, removal of unecessary vars!

**Pull request merged**